lib/util/index.js

```diff
@@ -718,7 +718,9 @@ function createMediaSignalingPayload(dominantSpeaker, networkQuality, trackPrior
     trackSwitchOff
       // eslint-disable-next-line
       ? { track_switch_off: transports }
-      : {}
+      : {},
+    // eslint-disable-next-line
+    { track_subscriptions: transports }
```

Comment:
Does the v3 to v2 downgrade imply that trackSwitchOff is no longer optional? The seamless downgrade is one of the reasons I proposed to make bandwidth profiles default / mandatory in the next major version.

In C++ SDK the track_switch_off is only negotiated only when a bandwidth profile is set today. If we make the API change the implementation would take care of itself I guess.

Reply:
@ceaglest Sure, I agree with making bandwidth profile mandatory (plus removing `disabled` for `trackSwitchOffMode`). However, I want to keep the bandwidth profile related changes for large rooms out of scope for March 1. I'm sure the react app always enables bandwidth profile right now (which will be used for the demo).

Reply:
Fair enough, as long as we don't expect the final behavior downgrade behavior in the pilot. 👍